app/controllers/internal/feedback_messages_controller.rb

```diff
@@ -3,7 +3,9 @@ class Internal::FeedbackMessagesController < Internal::ApplicationController
 
   def index
     @q = FeedbackMessage.
-      includes(:reporter, :notes).
+      includes(:reporter,
```

Comment:
why the three lines :D ?

Reply:
3 > 1

Reply:
To be honest, this is just how emacs thought it should look and I didn't want to argue. :grimacing:

Reply:
lol